tests/lib/rules/no-multi-comp.js

```diff
@@ -89,6 +89,21 @@ ruleTester.run('no-multi-comp', rule, {
     options: [{
       ignoreStateless: true
     }]
+  }, {
+    // multiple non-components
+    code: [
+      'import React, { createElement } from "react"',
+      'const helperFoo = () => {',
+      '  return true;',
+      '};',
+      'function helperBar() {',
+      '  return false;',
+      '};',
+      'function RealComponent() {',
+      '  return createElement("img");',
+      '};'
+    ].join('\n'),
+    parser: 'babel-eslint'
```

Comment:
The sourceType is required for anything using the default parser, but it's unnecessary for anything using babel-eslint.

Ideally, every test case that uses babel-eslint would also have a corresponding test case using the default parser, and babel-eslint tests would only be for features that eslint core can't handle (like flow types, or proposals)